Services/AssessmentQuestion/examples/class.exOfflinePresentationQuestionExporter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * For components that needs to integrate the assessment question service in the way,
+ * that questions act as independent assessment items client side, the former implementation
+ * is kept and supported by the ilAsqQuestion interface method toJSON.
```

Comment:
Please think about moving question_handling.js to the service and split it up per question

Reply:
I agree with you and I will provide additional changes soon. I think I wont get this done before the public workshop tomorrow.